### ISSUE TITLE
Add term soft tissue swelling

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10448]
**Preferred term label:**
Soft tissue

**Synonyms**
Soft tissue swelling (broad synonym)

**Definition (free text, please give PubMed ID)**
Abnormal swelling of soft tissue. This can be caused by, but is not limited to, trauma and inflammation 
Comment: if the swelling is due to fluid buidup the term 'edema' should be used.
PMID:24389367

**Parent term (use hpo.jax.org/app)**
Abnormality of connective tissue HP:0003549

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Caffey disease (omim114000)
Smithson S. (2019). Jürgen W. Spranger, Paula W. Brill, Christine Hall, Gen Nishimura, Andrea Superti-Furga, and Sheila Unger: Bone dysplasias: an atlas of genetic disorders of skeletal development: Fourth Edition, ISBN: 9780190626655. Human Genetics, 138(7), 787. https://doi.org/10.1007/s00439-019-02038-0

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Is there a definition that is not circular? I do not have access to the book, and it would be good to have an original PMID for this.
thanks!

**Comment by azankl** [comment]
@pnrobinson I think 'soft tissue swelling' is actually a broad synonym for `Edema HP:0000969`
'Edema' often implies a mechanism (like chronic venous insufficiency) while 'soft tissue swelling' is more descriptive, without suggesting a cause.

e.g. patients with Caffey disease (OMIM:114000, PMID: 24389367, PMCID: [PMC3987944](http://www.ncbi.nlm.nih.gov/pmc/articles/pmc3987944/)) characteristically develop soft tissue swelling that later disappears spontaneously, the cause is unclear. It's usually not referred to as edema in these cases.

Can we add 'soft tissue swelling' as a broad synonym to `Edema HP:0000969`?

Your nano-attribution (ORCID)
Andreas Zankl: 0000-0001-8612-1062
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
done!
